src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -434,8 +434,7 @@ public bool GetTypeInfo(Tizen.NUI.TypeInfo info)
         /// <since_tizen> 3 </since_tizen>
```

Comment:
Understood what you mean, but it's a public method. So, I think it would be better to add <remarks> or any description here, too.
What do you think?

Reply:
yes, you are absolutely right. I have added remark comment. thank you.